setup.py

```diff
@@ -5,10 +5,10 @@
 python_requires = '>=3'
 
 setup(
-    name='awscli-plugin-bolt',
-    packages=['awscli-plugin-bolt'],
+    name='awscli-plugin-granica',
+    packages=['awscli-plugin-granica'],
     version='2.0.4',
-    description='Bolt plugin for AWS CLI',
+    description='Granica plugin for AWS CLI',
     long_description=open('README.md').read(),
     long_description_content_type='text/markdown',
     author='Project N',
```

Comment:
I see url=https://gitlab.com/projectn-oss/projectn-bolt-awscli" below

perhaps we keep this folder/repo protected, and create a new

https://gitlab.com/projectn-oss/projectn-granica-awscli, with all the changes we are doing here?